yt_dlp/extractor/ntvde.py

```diff
@@ -34,40 +33,36 @@ def _real_extract(self, url):
         webpage = self._download_webpage(url, video_id)
 
         info = self._parse_json(self._search_regex(
-            r'(?s)ntv\.pageInfo\.article\s*=\s*(\{.*?\});', webpage, 'info'),
+            r'(?s)article:\s*(\{.+?\})', webpage, 'info'),
             video_id, transform_source=js_to_json)
```

Comment:
Can be simpified:
```python
info = self._search_json(
    r'article:\s*', webpage, 'info', video_id, transform_source=js_to_json)
```

Reply:
Thanks for the suggestion, I applied it.
This is my first pull request for the project, and I'm not really familiar with all the available helper methods like `_search_json()`